udata_front/theme/gouvfr/assets/less/content/typography.less

```diff
@@ -318,4 +322,15 @@ each(@breakpoints, .(@bk, @_k, @_i) {
     margin-bottom: auto;
     color: @grey-380;
     font-size: 0.6rem;
-  }
+}
+
+.dash-before-sm::before {
```

Comment:
Can you add the class to the rule above ? It should work with `.dash-after-sm::after, .dash-before-sm::before` no ? Also, I don't know why I didn't do it for `.dash-before::before` and `.dash-after::after`.

Reply:
Fixed in 4fc79013